client/src/components/Ratings/sub-components/UploadImageModal.jsx

```diff
@@ -44,76 +43,72 @@ export default function UploadAndDisplayImage({
           {selectedImage.length < 5 && (
-            <>
-              <div
-                className="drag-and-drop"
-                onDragOver={(e) => e.preventDefault()}
-                onDrop={(e) => {
-                  e.preventDefault();
-                  const { files } = e.dataTransfer;
-                  // const files = e.dataTransfer.files;
-                  if (files.length) {
-                    setSelectedImage([...selectedImage, files[0]]);
-                    handleSubmit(files[0]);
-                  }
-                }}
+            <div
+              className="drag-and-drop"
+              onDragOver={(e) => e.preventDefault()}
+              onDrop={(e) => {
+                e.preventDefault();
+                const { files } = e.dataTransfer;
+                // const files = e.dataTransfer.files;
+                if (files.length) {
+                  setSelectedImage([...selectedImage, files[0]]);
+                  handleSubmit(files[0]);
+                }
+              }}
+            >
+              <p
+                className="drag-and-drop-text"
               >
-                <p
-                  className="drag-and-drop-text"
-                >
-                  Drag And Drop Or Click To Upload Your Image!
-                  <FcUpload className="review-image-upload-icon" />
-                  <input
-                    className="xyz"
-                    type="file"
-                    name="myImage"
-                    onChange={(event) => {
+                Drag And Drop Or Click To Upload Your Image!
+                <FcUpload className="review-image-upload-icon" />
+                <input
+                  className="xyz"
+                  type="file"
+                  name="myImage"
+                  onChange={(event) => {
+                    if (event.target.files[0]) {
                       setSelectedImage([...selectedImage, event.target.files[0]]);
                       handleSubmit(event.target.files[0]);
-                    }}
-                  />
-                </p>
-              </div>
-              {/* <input
-                className="review-image-main-upload-button"
-                type="file"
-                name="myImage"
-                onChange={(event) => {
-                  setSelectedImage([...selectedImage, event.target.files[0]]);
-                  handleSubmit(event.target.files[0]);
-                }}
-              /> */}
-            </>
+                    }
+                  }}
+                />
+              </p>
+            </div>
           )}
         </div>
-        <h1 className="write-review-thumbnail">Show us your look!</h1>
+        <h1 className="write-review-thumbnail" />
         {selectedImage
           ? selectedImage.map(
             (image, index) => (
-              <div key={image} className="write-review-thumbnail">
+              <div key={image} className="write-review-thumbnail" style={{ height: '250px' }}>
```

Comment:
Is that possible to add the style into class, something like
div  .write-review-thumbnail?

Reply:
Removed inline styling